tests/karate/docs/README.md

```diff
@@ -0,0 +1,172 @@
+
+<div align="center">
+  <img src="images/architecture_diagram.png" alt="Architecture"/>
+</div>
+
+### Example: Create a valid roll call scenario
+
+```
+Feature: Create a Roll Call
+
+  Background:
+
+    * call read('classpath:be/features/utils/constants.feature')
+    * call read(serverFeature)
+    * call read(mockClientFeature)
+    * def organizer = call createMockFrontend
+    * def lao = organizer.createValidLao()
+    * def validRollCall = organizer.createValidRollCall(lao)
+    * call read(createLaoScenario) { organizer: '#(organizer)', lao: '#(lao)' }
+
+  @createRollCall1
+  Scenario: Valid Roll Call
+    Given def validCreateRollCall =
+      """
+        {
+          "object": "roll_call",
+          "action": "create",
+          "id": '#(validRollCall.id)',
+          "name": '#(validRollCall.name)',
+          "creation": '#(validRollCall.creation)',
+          "proposed_start": '#(validRollCall.start)',
+          "proposed_end": '#(validRollCall.end)',
+          "location": '#(validRollCall.location)',
+          "description": '#(validRollCall.description)',
+        }
+      """
+    When organizer.publish(validCreateRollCall, lao.channel)
+    And json answer = organizer.getBackendResponse(validCreateRollCall)
+    Then match answer contains VALID_MESSAGE
+    And match organizer.receiveNoMoreResponses() == true
+```
+
+## First Setup
+
+### All
+Make sure that you have [maven](https://maven.apache.org/download.cgi) installed.
+The Karate plugin for IntelliJ is also recommended.
+
+### Backend
+
+To test a backend, you don't need more setup that what is needed to build that backend. Use the resources provided by
+those projects.
+
+### Android Front-end
+
+To test the Android Frontend, you need to have an Android emulator installed. The easiest way to achieve it is to
+install it through [Android Studio](https://developer.android.com/studio) :
+Go to `Tools -> AVD Manager` and create an emulator.
+
+Then you need to install Appium. You can install either the command line app with `npm install -g appium`, or
+the [Desktop App](https://github.com/appium/appium-desktop/releases/).
+
+Finally, you need to set the environment variable `ANDROID_HOME` (The previous name was`ANDROID_SDK_ROOT`
+and it still works)
+to your Android SDK installation. Find it by
+opening Android Studio and going to `Tools -> SDK Manager`. It stands next to `Android SDK Location`.
+
+If your Computer runs on Windows : we strongly advise that you do not use a VM or WSL. You will encounter problems you
+would not have otherwise, some of which might even be technically impossible to solve.
+
+### Web Front-end
+
+Make sure you have [Google Chrome](https://www.google.com/intl/en/chrome/) and [npm](https://nodejs.org/en/download/)
+installed.
+
+## Running the Tests
+
+### Backend
+
+Build the backend you want to test. Follow the steps described in the corresponding subproject.
```

Comment:
How does Karate find the built executables ? Does it need to be copied over ? 🤔 
I don't remember this part.